packages/worker/src/index.ts

```diff
@@ -194,12 +195,12 @@ const pollTranscriptionQueue = async (
 
 	const maybeSentTimestamp: string | undefined | null =
 		message.message.Attributes?.SentTimestamp;
-	const enqueuedAtEpochMillis =
+	const maybeEnqueuedAtEpochMillis =
```

Comment:
was going to suggest starting the clock in the [api endpoints](https://github.com/guardian/transcription-service/blob/03238eac1ca66c789c64cc72ac8161c8c9327b5b/packages/api/src/index.ts#L144) that do the enqueueing but the way you've done it also works for transcription jobs from Giant which write directly to the queue so I think your way is better